spec/requests/articles/articles_show_spec.rb

```diff
@@ -15,31 +17,64 @@
       expect(response).to have_http_status(:ok)
     end
 
-    it "renders the proper title" do
-      expect(response.body).to include CGI.escapeHTML(article.title)
-    end
-
-    it "renders the proper published at date" do
-      expect(response.body).to include CGI.escapeHTML(article.readable_publish_date)
-    end
-
-    # This is a flakey spec. Due to how we are localizing date times, sometimes
-    # the date is off. See https://github.com/thepracticaldev/dev.to/issues/7086.
-    xit "renders the proper modified at date" do
-      article.update(edited_at: Time.zone.now)
-      get article.path
-      expect(response.body).to include GI.escapeHTML(article.edited_at.strftime("%b %d, %Y"))
-    end
-
-    it "renders the proper author" do
-      expect(response.body).to include CGI.escapeHTML(article.cached_user_username)
+    # rubocop:disable Rspec/ExampleLength
```

Comment:
the `S` is capital, just telling you because rubocop flagged this in another PR, see https://github.com/thepracticaldev/dev.to/pull/7743/files#diff-ba9ab82dc4668287786466b0edda73ffR18 :D

Reply:
Thank you for pointing that out! I've made this change. 😄